api/index.js

```diff
@@ -19,4 +20,41 @@ var API = (function(userFound) {
   }
 })(user)
 
-module.exports = API
+var DB = (() => {
+  const validateCode = (code) => {
+    // validates access code
+    code = code.toUpperCase()
+
+    // Real DB code will go here
+    let dbUser = false;
+    dbUser = db.find(function(u, i) {
+      if(u.code === code)
+        return u
+    })
+    // return the whole row for now for populating session with initial data
+    return dbUser
+  }
+
+  const validateUser = (login) => {
+    // validates user code / sin / DoB
+    let dbUser = false
+    
+    dbUser = db.find(function(u,i) {
```

Comment:
This is a minor thing, but could we explicitly make it `user` instead of `u`. It also looks like we're not using the index  (i) anywhere (in this find and also not in the find on line 30), so no need to pass it. Also for consistency, how do you feel about changing this to use the arrow function (unless there's reason not to)? So:
```
db.find( user => {
  //do stuff here
})
```

Reply:
yup 👍